octavia-cli/octavia_cli/generate/templates/connection.yaml.j2

```diff
@@ -18,5 +18,30 @@ configuration:
   schedule: # OPTIONAL | object
     time_unit: hours # REQUIRED | string | Allowed values: minutes, hours, days, weeks, months
     units: 1 # REQUIRED | integer
+  {%- if enable_normalization %}
+  operations:
+    - name: "Normalization"
+      operator_configuration:
+        normalization:
+          option: "basic"
+        operator_type: "normalization"
```

Comment:
Not all destinations support normalization, so I think it will be confusing to allow normalization to be enabled by the CLI user when creating any connection, as it will make them think that any connection being created can support normalization.

Instead, I think it would be better if we refactored these changes in the following ways:
1. Remove the `--normalization` flag from the `generate connection` command, because normalization is not supported for all destinations
2. When the CLI is generating the connection yaml file, it should first check if the destination supports normalization. If so, then the commented block that you have below (i.e. the `Uncomment and edit the block below if you want to enable Airbyte normalization` /  `Uncomment and edit the block below if you want to declare a custom transformation` blocks) should be included in the generated yaml file.

This way, the user doesn't need to decide whether or not to enable normalization - instead, the CLI will automatically include those normalization configuration blocks in the generated yaml file _if_ the destination being used in the connection supports normalization, leaving it up to the user to modify those blocks as they wish if they want to use normalization.

Reply:
Good call! Thank for the suggestion and the reminder, I will definitely implement this.

Reply:
I implemented your suggestion: 
* A commented `operations` field is added to connection configurations having destination supporting transformations
* No `operations` field is added when the destination does not support transformation.